.gitignore

```diff
@@ -73,3 +73,4 @@ target/
 .ipynb_checkpoints
 
 .vscode/
+.idea/
```

Comment:
I think this might be removed.

Reply:
I added this as I saw the `.vscode` settings were included and thought it might be worthwhile adding `.idea` (PyCharm) settings too. Happy to remove it though, if you'd prefer 🙂

Reply:
Sorry i didnt know that. It will be fine i think.